feat: add launch image iOS

SputNikPlop opened the pull request:
This adds a launch image to the app instead of keeping it entirely blank on startup.

SputNikPlop committed:
feat: add launch image

SputNikPlop commented:
The app appears to have a gray box around the logo. Would it be possible @MateusRodCosta to modify the logo to make it whiter so it appears more natural?
<img width="340" alt="Screenshot 2023-08-28 at 22 33 17" src="https://github.com/MateusRodCosta/vidya_music/assets/100245448/00e92a2b-44c0-487d-a7bd-7dc2b58cb33f">

MateusRodCosta commented:
> The app appears to have a gray box around the logo. Would it be possible @MateusRodCosta to modify the logo to make it whiter so it appears more natural? <img alt="Screenshot 2023-08-28 at 22 33 17" width="340" src="https://user-images.githubusercontent.com/100245448/263911198-00e92a2b-44c0-487d-a7bd-7dc2b58cb33f.png">

This is likely because I use #fafafa as the background color instead of #ffffff.
I think you will have to recreate the icon with transparent background from the svgs at the misc folder.

MateusRodCosta commented:
Hey, I took a quick look at your PR and apparently it focus only on iOS?

> The app appears to have a gray box around the logo. Would it be possible @MateusRodCosta to modify the logo to make it whiter so it appears more natural?

I updated the icons just now on the dev branch, you will likely want one of the pngs that has either transparent or white background. The Google Play icon might also be an option since it adds scales down the icon to add some padding around it.

Do note that while you are handling iOS splash screen, there's also the need to do something for pre-Android 12.Android 12+ is optional since there is a default splash screen implementation.

In anyway, please create new PRs on top of the dev branch in the future.

SputNikPlop committed:
fix: launch images

SputNikPlop commented:
> I updated the icons just now on the dev branch, you will likely want one of the pngs that has either transparent or white background. The Google Play icon might also be an option since it adds scales down the icon to add some padding around it.

I updated the PR with this, will try android and to the correct branch.

SputNikPlop committed:
Merge branch 'dev' into icon

SputNikPlop committed:
Merge branch 'dev' into icon

SputNikPlop committed:
Merge branch 'dev' into icon
